de.ient.YUView.yaml

```diff
@@ -0,0 +1,31 @@
+app-id: de.ient.YUView
```

Comment:
It does not seem to load a website here. Just to make sure, do you own this domain?

Reply:
Nope. The docs say
`
Flatpak identifies each application and runtime using a unique three-part identifier, such as com.company.App
`
I didn't realize this should correspond to a website. Does it have to? The corresponding websites would be https://github.com/IENT/YUView and http://www.ient.rwth-aachen.de. The latter is our institutes website and doesn't mention this software anywhere.

Reply:
I talked with @TingPing and in general we prefer something you control, with exceptions for git hosting services like GitHub or GitLab. The domain doesn't really need to mention the project, so whatever suits you best.

Reply:
can it have more than 3 parts? i found these others:
https://github.com/flathub/com.github.bajoja.indicator-kdeconnect
https://github.com/flathub/com.github.marktext.marktext

also there does not seem to be a website for example at https://marktext.marktext.github.com/

I think we could do de.ient.rwth-aachen.YUView and de.ient.rwth-aachen.RDPlot for the other tool I'm still trying to package

Reply:
Yes, 3 is minimum, but it can have more. Looks good; let's change it, run final test build and merge it!

Reply:
alright! should I create a new branch with that name or do the changes in this one?

Reply:
wait, should it be 
de.ient.rwth-aachen.YUView 
or 
de.rwth-aachen.ient.YUView
?

Reply:
de.rwth-aachenient.YUView

Just rename the files and change the appid where needed, branch can stay the same. Thanks!

Reply:
now I have this problem:
`
error: 'de.rwth-aachen.ient.YUView' is not a valid application name: Only last name segment can contain -
`

Reply:
Right, you'll have to replace the dash with an undescore.

Reply:
should be finished now :)